wandb/proto/wandb_telemetry.proto

```diff
@@ -116,6 +116,17 @@ message Imports {
   bool stability_sdk = 88;
   bool prefect = 89;
   bool prefect_ray = 90;
+  bool pinecone = 91;
```

Comment:
Can you add a comment after the line that tells what the name of the pypi package is called if it is different (this is information that we got from the tests but since those have fallen out of style, lets do this instead)

```
   bool pinecone = 91;  // pinecone-client
```